src/api/get.js

```diff
@@ -0,0 +1,39 @@
+const URL = 'http://localhost:3000'
```

Comment:
This is hardcoded in this file for now, but will ideally be in a shared file somewhere.